### PR TITLE
Update Italian translation and sort baseline strings lexicographically

### PR DESCRIPTION
**Pull request by andreaippo**
Update Italian translation:
- remove capitalization where unnecessary
- translate new strings
- minor changes
- sort lexicographically

Baseline strings file:
- sort lexicographically. This should make it easier for translators to rebase their files, as it ensures a consistent ordering between the baseline file and the translation. Alternatively, core devs should always append new strings at the end of the file, but I think that sorting makes it look tidier. Let me know what you think.

### CODE REVIEW

**Comment by twizmwazin** [comment]
I'm in favor of lexicographic sorting, especially if we can enforce it with CI.

**Comment by andscape-dev** [comment]
By the way, my local build is failing. "Unresolved reference: ic_launcher_foreground" in `MarkdownHelper.kt` and `PictrsImage.kt`. Seems unrelated, but is this just my local env being broken or is it happening to you too?

**Comment by dessalines** [comment]
@andscape-dev I had this once, found the fix on stackoverflow. Clean gradle cache and then run invalidate caches and restart in android studio, iirc.

@twizmwazin @andreaippo We use a prettier CI image to check our json and a lot more with all our lemmy projects. Looks like it has an [xml plugin](https://github.com/prettier/plugin-xml). I'm just not sure it would be very desirable tho, because every little translation addition would require us to manually fix the formatting. Contributers aren't going to install prettier and run it for us.

**Comment by andscape-dev** [comment]
@andreaippo AFAIK Android Studio doesn't have a feature to sort XML entries by tag value, and it is the standard IDE for Android. We can't force all contributors to use Android Studio, but we can assume most of them will.

To me, keeping entries in the order they are added to the codebase seems the simplest solution. That way changes willl be clustered towards the bottom and therefore easier to review/merge.

**Comment by andscape-dev** [comment]
@andreaippo have you had time to take a look at my comments? They're minor suggestions, we can merge without them too...

**Comment by andreaippo** [comment]
> @andreaippo have you had time to take a look at my comments? They're minor suggestions, we can merge without them too...

Hey, I can't seem to find any comments. Are they in the diff somewhere?

**Comment by andscape-dev** [comment]
> Hey, I can't seem to find any comments. Are they in the diff somewhere?

Weird, I see them here in the thread as an open review...

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -1,282 +1,291 @@
...
+    <string name="login_view_model_incorrect_login">Login errato</string>
+    <string name="login_view_model_is_not_a_lemmy_instance">%1$s non è un\'istanza Lemmy</string>
+    <string name="look_and_feel_activity_show_content_for_collapsed_comments">Mostra contenuto per commenti minimizzati</string>
+    <string name="look_and_feel_font_size">Dimensione carattere: %1$s</string>
+    <string name="look_and_feel_look_and_feel">Aspetto</string>
+    <string name="look_and_feel_post_view">Formato post</string>
+    <string name="look_and_feel_post_view_card">Scheda</string>
+    <string name="look_and_feel_post_view_list">Lista</string>
+    <string name="look_and_feel_post_view_small_card">Scheda piccola</string>
+    <string name="look_and_feel_show_action_bar_for_comments">Mostra barra delle azioni per i commenti</string>
```
Comment (andscape-dev) [inline app/src/main/res/values-it/strings.xml:183]: "Mostra sempre la barra delle azioni per i commenti"?

If this setting isn't active you can still see the action bar, you just have to long-press the comment body... Can we communicate this better without too long a sentence?

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -1,282 +1,291 @@
...
     <string name="private_message_reply_back">Indietro</string>
-    <string name="private_message_reply_type_your_message_placeholder">Digita il tuo messaggio</string>
-    <string name="create_report_report">Segnala</string>
-    <string name="create_report_back">Indietro</string>
-    <string name="create_report_type_your_reason">Digita la motivazione</string>
+    <string name="private_message_reply_reply">Rispondi</string>
+    <string name="private_message_reply_type_your_message_placeholder">Digita il messaggio</string>
+    <string name="private_message_to">"a "</string>
+    <string name="removeBookmark">Rimuovi dai segnalibri</string>
+    <string name="selectSort">Scegli criterio ordinamento</string>
```
Comment (andscape-dev) [inline app/src/main/res/values-it/strings.xml:258]: "Scegli il criterio di ordinamento"?

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -1,282 +1,291 @@
...
-    <string name="settings_about_version">Versione %1$s</string>
-    <string name="settings_about_support">Supporto</string>
-    <string name="settings_about_issue_tracker">Segnalazione problemi</string>
     <string name="settings_about_developer_matrix_chatroom">Chat Matrix degli sviluppatori</string>
     <string name="settings_about_donate_to_jerboa_development">Sostieni lo sviluppo di Jerboa</string>
-    <string name="settings_about_join_c_jerboa">Partecipa a c/jerboa</string>
     <string name="settings_about_follow_on_mastodon">Seguici su Mastodon</string>
+    <string name="settings_about_issue_tracker">Segnalazione problemi</string>
+    <string name="settings_about_join_c_jerboa">Partecipa a c/jerboa</string>
     <string name="settings_about_open_source">Open Source</string>
```
Comment (andscape-dev) [inline app/src/main/res/values-it/strings.xml:265]: Is Open Source capitalized here on purpose?

**Comment by andscape-dev** [comment]
So yeah, I have no idea why, but Github was showing my review to me only, and not to anyone else...

**Comment by andreaippo** [comment]
> So yeah, I have no idea why, but Github was showing my review to me only, and not to anyone else...

Happened to me all the time when I first started using Gitlab after a long period on BitBucket. In my case I had to submit/finalize my review, otherwise comments would stay private :D

**Comment by andscape-dev** [comment]
Awesome, LGTM!

**Comment by andreaippo** [comment]
> Awesome, LGTM!

Great!

@twizmwazin  @dessalines we are ready for merge :)

**Pull request merged**